Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,63 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            //Equation : a * b = c
+
+            string a = equation.Substring(0, equation.IndexOf('*'));
+            string b = equation.Substring(equation.IndexOf('*') + 1, equation.IndexOf('=') - equation.IndexOf('*') - 1);
+            string c = equation.Substring(equation.IndexOf('=') + 1);
+
+            if (a.Contains('?'))
+            {
+                int result = (int.Parse(c)) / int.Parse(b);
+                if (result * int.Parse(b) == int.Parse(c))
+                {
+                    string strResult = result.ToString();

```

Comment:
the same logic is repeated in 3 if else. You can create one single method so that code will not get repeated.